pkg/operator/ceph/cluster/status/conditions.go

```diff
@@ -0,0 +1,300 @@
+/*
+Copyright 2016 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package status to provide conditions for CephCluster
+package status
```

Comment:
How about adding condition.go to `pkg/operator/ceph/config`? This seems like it could belong there rather than creating a new package.

Reply:
Does it. Okay then. I will put this inside the `/config`.